libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Message.cs

```diff
@@ -26,14 +26,15 @@ public class Message
         /// <summary>
         /// Gets a list of attachments.
         /// </summary>
-        /// <value>Attachments.</value>
+        /// <value>Attachments that could come with a Facebook message.</value>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "it needs to be set to null in SendActivitiesAsync")]
```

Comment:
Make this collection readonly and remove the suppression - follow same pattern as the other collections from yesterdays changes.